Makefile

```diff
@@ -223,6 +223,13 @@ $(IMAGE_TAR_GZ):
 	@echo ">>> Creating tar for image..."
 	docker save $(IMAGE_NAME):latest | gzip > $(IMAGE_TAR_GZ)
 
+# this target should not be necessary: Go 1.11 will download thihngs on demand
+# however, this is useful in Circle-CI as the dependencies are already cached
+# this will craete the `vendor` from that cache and pass them to the `docker build`
```

Comment:
Minor: typo `craete`

Reply:
oh you are the typo finder master :cupid: